scripts/component_hash_update/src/component_hash_update/components.py

```diff
@@ -88,7 +88,7 @@
         "graphql_id": "R_kgDOHQ6J9w",
     },
     "youki": {
-        "url": "https://github.com/youki-dev/youki/releases/download/v{version}/youki-{version}-{alt_arch}-gnu.tar.gz",
+        "url": "https://github.com/youki-dev/youki/releases/download/v{version}/youki-{version}-{alt_arch}-musl.tar.gz",
```

Comment:
Not sure why, but it looks like this is not the url we use for the download.
 "{{ github_url }}/containers/youki/releases/download/v{{ youki_version }}/youki-{{ youki_version }}-{{ ansible_architecture }}-musl.tar.gz" (containers/youki). Not sure why we diverged ?